client/devdocs/docs-example/readme-viewer.jsx

```diff
@@ -19,21 +19,25 @@ class ReadmeViewer extends Component {
 	static propTypes = {
 		getReadme: PropTypes.func,
 		readmeFilePath: PropTypes.string,
+		section: PropTypes.string,
 	};
 
 	static defaultProps = {
-		getReadme: readmeFilePath => {
+		getReadme: ( section, readmeFilePath ) => {
```

Comment:
Seems like this could be simplified with a default value for `section`:

```js
getReadme: ( section = 'components', readmeFilePath ) => {
	return htmlToReactParser.parse( require( `../../${ section }/${ readmeFilePath }/README.md` ) );
}
```

Reply:
Oh wait, I just realised `section` might not work like that. But maybe:

```js
getReadme: ( section, readmeFilePath ) => {
	const sectionPath = section === 'blocks' ? 'blocks' : 'components';
	return htmlToReactParser.parse( require( `../../${ sectionPath }/${ readmeFilePath }/README.md` ) );
}
```